src/config.ts

```diff
@@ -0,0 +1,31 @@
+import SdkConfig from "./SdkConfig";
+
+const config = SdkConfig.get();
+
+const hiddenSplashScreenMenuItem = config['hiddenSplashScreenMenuItem'];
+
+export const signInTextNeedsToBeReplaced = hiddenSplashScreenMenuItem?.find(x => x.includes(['changeSigninWithLoginTextLabel'])) ? true: false;
+
+// background color on login screen
+// used by app-web to load a colored background instead of lake image
+export const splashScreenBackgroundColor = hiddenSplashScreenMenuItem?.find(x => x['splashScreenBackgroundColor']) || null;
+
+// Sign in text - change to login
+export const changeSigninWithLoginTextLabel = () => {
+    console.log('change sign in text', signInTextNeedsToBeReplaced);
+    if (signInTextNeedsToBeReplaced) {
+        document.querySelector('label').innerHTML = document.querySelector('label').innerHTML.replace(/Sign in/g, "Log in");
+        document.querySelector('input').innerHTML = document.querySelector('input').innerHTML.replace(/Sign in/g, "Log in");
+        document.querySelector('h2').innerHTML = document.querySelector('h2').innerHTML.replace(/Sign in/g, "Log in");
+     //   document.querySelector('form').innerHTML = document.querySelector('.mx_SSOButtons').innerHTML.replace(/Sign in/g, "Log in");
+    }
+}
+
+//Phone based authentication drop down menu
+export const allowPhoneBasedAuthentication = hiddenSplashScreenMenuItem?.find(x => x.includes(['allowPhoneBasedAuthentication'])) ? false: true;
```

Comment:
Does this return true if hiddenSplashScreenMenuItem doesn't exist?

Reply:
Yeah, if you would remove hiddenSplashScreenMenuItem array from config then screen would return to normal which is how I had checked reverse so that default behavior stays on.

Reply:
Explanation has been added now in  Kanban, feature doc, and on config.ts.